app/components/helpers/HelpButton.tsx

```diff
@@ -66,7 +67,25 @@ const HelpButton: React.FunctionComponent<Props> = ({
               </li>
             </ul>
           ) : (
-            ''
+            <ul>
+              <li>
+                <span role="img" aria-label="open book emoji">
+                  📖
+                </span>
+                <a href={docsUrl} target="_blank" rel="noopener noreferrer">
+                  See the help documentation
+                </a>
+              </li>
+              <li>OR</li>
+              <li>
+                <span role="img" aria-label="letter emoji">
+                  ✉️
+                </span>
+                <Link href="/resources/contact">
```

Comment:
It would probably be best for this to open in a new tab so it doesn't force them to navigate away (in case they're in the middle of an application, for example).

You can use the Link component to wrap an `<a>` link tag in order to specify the `target` and `rel` attributes as above.

Reply:
FYI, I believe that a Link should always have an `<a>` element (or a component that renders an `<a>` element) as a child, otherwise a warning will be printed in the console (I couldn't find why in the nextjs docs)

Reply:
The Footer element doesn't use `<a>` I think it's more if you need to override something on the a tag like target etc?